docs/user-guide/cli-authentication-methods.md

```diff
@@ -0,0 +1,288 @@
+        }
+   ``` 
+
+:::note
+
+If you have multiple types of configuration files and base profiles, see [How configuration files and profiles work together](../user-guide/cli-using-understand-profiles-configs.md#how-configuration-files-and-profiles-work-together) to learn which configuration and profile would be used to store the API ML token.
+
+:::
+
+### Logging out
+
+Log out to remove the token from your base profile and prompt the API ML to invalidate the token:
+
+```
+zowe auth logout apiml
+```
+
+This command sends a request to the API ML to invalidate the token. Log in again to obtain a new token.
+
+If you used the `--show-token` option and never stored your token in profile, add `--token-value <123>` (where `<123>` is the value of the token) to this command to invalidate the token. 
+
+### Accessing a service through API ML
+
+To access mainframe services with SSO through API ML using the token in your base profile, use the `basePath` property in your service profile to indicate the base path of the API on the API ML instance that you want to access.
+
+To establish a base path, see instructions for [Zowe team configuration](#specifying-a-base-path-with-zowe-team-configuration).
+
+:::note
+
+Ensure that you *do not* provide username, password, host, or port directly on the service commands or profiles. Supplying those options causes the CLI to ignore the API ML token in your base profile and access the service directly.
+
+:::
+
+#### Specifying a base path with Zowe team configuration
+
+Use the following steps to specify a base path with Zowe [team configuration](../appendix/zowe-glossary.md#team-configuration):
+
+1. Note the complete path for a service registered to API ML.
+
+    For example:
+
+    ```
+    https://myapilayerhost:port/<yourServiceId>/api/v1
+
+    ```
+
+    The format of base paths can vary based on how API ML is configured at your site.
+
+2. Using the example in Step 1, access the API ML instance by creating or updating a [service profile](../user-guide/cli-using-using-team-profiles.md#zowe-cli-profile-types), or issuing a command, with the base path value of `<yourServiceId>/api/v1`. Your service profile uses the token and credentials stored in your default base profile.
+
+    To create or update a service profile with the preceding base path in a **project** team configuration file:
+
+    ```
+    zowe config set profiles.<yourService>.properties.basePath <basePath> 
+    ```
+
+    If you are using a **global** team configuration file (located in your home directory), add `--global-config` to the end of the command.
+
+    Commands issued with this profile are routed through API ML to access an appropriate instance of your service.
+
+#### Specifying a base path with Zowe V1 profiles
+
+See the [Integrating with API Mediation Layer](https://67c89aa5af702da5881fc564--zowe-docs-master.netlify.app/v1.28.x/user-guide/cli-usingcli#integrating-with-api-mediation-layer) Zowe V1 documentation.
```

Comment:
This seems like V1 specific doc, should it be removed?

Reply:
Can't they look at the V2.18 docs for that?

Reply:
I think we've talked about this during a meeting and decided to leave in links to archived content.